src/App.jsx

```diff
@@ -22,28 +22,15 @@ export function App() {
 	}
 
 	useEffect(() => {
-		if (listToken) navigate('/list');
+		if (listToken) {
+			navigate('/list');
 
-		// }
-		/**
-		 * streamListItems` takes a `listToken` so it can communicate
-		 * with our database; then calls a callback function with
-		 * a `snapshot` from the database.
-		 *
-		//  * Refer to `api/firebase.js`.
-		//  */
-		// 	return streamListItems(listToken, (snapshot) => {
-		// 		/**
-		// 		 * Read the documents in the snapshot and do some work
-		// 		 * on them, so we can save them in our React state.
-		// 		 *
-		// 		 * Refer to `api/firebase.js`
-		// 		 */
-		// 		const nextData = getItemData(snapshot);
+			return streamListItems(listToken, (snapshot) => {
+				const nextData = getItemData(snapshot);
 
-		// 		/** Finally, we update our React state. */
-		// 		setData(nextData);
-		// 	});
+				setData(nextData);
+			});
+		}
```

Comment:
Victoria and I moved this part from App.jsx to List.jsx because the List componente is fetching the list items:

useEffect(() => {
		streamListItems(listToken, (snapshot) => {
			const nextData = getItemData(snapshot);

			setData(nextData);
			setLoading(false);
		});
	}, [listToken]);